change/@ni-nimble-components-c20d1978-2348-4144-866b-63c75566bf94.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
It doesn't have to be in this PR, but we'll need to add Angular and Blazor integration to expose the new `appearance` attribute.

Reply:
Just added integration for both in my most recent commit 😄